src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.DukeList;
+import duke.UserInterface;
+import duke.exception.InvalidArgumentException;
+import duke.task.Task;
+
+/**
+ * Represents a command for which marks a task in the main dukelist as
+ * completed.
+ */
+public class DoneCommand implements Command {
+    private final DukeList dukeList;
+    private final UserInterface ui;
+
+    /**
+     * Creates a command that marks a task from the given dukelist as completed, and
+     * provide its response to the given ui.
+     *
+     * @param dukeList list of tasks from which the task will marked as completed
+     * @param ui       user interface for displaying responses
```

Comment:
It seems that there is an unnecessary space here.

Reply:
Thanks for the review! It is a linter that auto-formats the param description to auto-align itself. I thought it was ok to leave it as such, but thanks anyways!